code/game/machinery/portable_turret.dm

```diff
@@ -733,35 +622,30 @@ Status: []<BR>"},
 	if (!raised) // the turret has to be raised in order to fire - makes sense, right?
 		return
 
-
-	// any emagged turrets will shoot extremely fast! This not only is deadly, but drains a lot power!
-
-	if(iconholder)
+	if(!emagged)
 		icon_state = "[lasercolor]target_prism"
 	else
 		icon_state = "[lasercolor]orange_target_prism"
 
+	use_power(reqpower)
+
+//		 //Shooting Code:
```

Comment:
It would be better if, rather than copying projectile firing code, it instead simply fired the gun itself
But I'm not sure exactly how a gun behaves when fired without a valid `user`

Making it not care shouldn't be too difficult though